pyproject.toml

```
@@ -1,3 +1,186 @@
+  "urllib3>=1.26.5, <2.0",  # 1.26.16 / 2.0.4
+]
+
+[project.optional-dependencies]
+test-convert = [
+  "html2text>=2020.1.16",
+  "markdown>=3.3.4",
+  "nitrate>=1.8.2",
+  "python-bugzilla>=3.2.0",
+]
+export-polarion = [
+  "pylero>=0.0.8",
+]
+provision-beaker = [
+  "mrack>=1.15.1",
+]
+provision-virtual = [
+  "testcloud>=0.9.10",
+]
+provision-container = []
+report-junit = [
+  "junit_xml>=1.9",
+]
+report-polarion = [
+  "tmt[report-junit]",
+  "tmt[export-polarion]",
+]
+all = [
+  "tmt[test-convert]",
+  "tmt[export-polarion]",
+  "tmt[provision-container]",
+  "tmt[provision-virtual]",
+  "tmt[provision-beaker]",
+  "tmt[report-junit]",
+  "tmt[report-polarion]",
+]
+# Needed for readthedocs and man page build. Not being packaged in rpm.
+docs = [
+  "sphinx_rtd_theme>=1.2.2",  # The theme pulls a specific sphinx version
+  "sphinxcontrib-manpage>=0.6",
+]
+
+[project.scripts]
+tmt = "tmt.__main__:run_cli"
+
+[project.urls]
+Homepage = "https://github.com/teemtee/tmt"
+
+[tool.hatch.version]
+source = "vcs"
+
+[tool.hatch.build.hooks.vcs]
+version-file = "tmt/_version.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+  "/tmt",
+  "/LICENSE",
+  "/examples",
+  "/README.rst",
+  "/tests",
+  "/tmt.spec",
+  "/tmt.1"
+]
+
+[tool.hatch.envs.default]
+platforms = ["linux"]
+
+[tool.hatch.envs.dev]
+description = "Development environment"
+dependencies = [
+  "typing-extensions>=4.4.0; python_version < '3.10'",  # TypeAlias introduced in 3.10, Self in 3.11
+  "autopep8",
+  "ruff",
+  "mypy",
+  "pre-commit",
+  "types-Markdown",
+  "types-requests",
+  "types-setuptools",
+  "types-jsonschema",
+  "types-urllib3",
+  "types-jinja2",
+  "types-babel",
+]
+features = [
+  "test-convert",
+  "export-polarion",
+  "provision-beaker",
+  "provision-container",
+  "provision-virtual",
+  "report-junit",
+  "report-polarion",
+]
+
+[tool.hatch.envs.dev.scripts]
+lint = ["autopep8 {args:.}", "ruff --fix {args:.}"]
+type = ["mypy {args:tmt}"]
+check = ["lint", "type"]
+
+[tool.hatch.envs.test]
+description = "Testing environment"
+system-packages = true  # Install python3-nitrate to skip building psycopg2
+dependencies = [
+  "pytest",
+  "python-coveralls",
+  "requre",
+  "yq==3.1.1",
+]
+features = [
+  "test-convert",
+  "report-junit",
+]
+
+[tool.hatch.envs.test.scripts]
+unit = "pytest -vvv -ra --showlocals tests/unit"
+smoke = "pytest -vvv -ra --showlocals tests/unit/test_cli.py"
+cov = [
+  "coverage run --source=tmt -m pytest -vvv -ra --showlocals tests",
+  "coverage report",
+  "coverage annotate",
+]
+requre = [
+  "cd {root}/tests/integration",
+  "pytest -vvv -ra --showlocals",
+  "requre-patch purge --replaces :milestone_url:str:SomeText --replaces :latency:float:0 tests/integration/test_data/test_nitrate/*",
+]
+
+[[tool.hatch.envs.test.matrix]]
+python = ["3.9", "3.11"]
+
+[tool.hatch.envs.docs]
+dependencies = ["tmt[docs]"]
+
+[tool.hatch.envs.docs.scripts]
+create = "cd {root}/docs && make html"
```

```suggestion
html = "cd {root}/docs && make html"
```

Perhaps? To make consistent `hatch run docs:man` and `hatch run docs:html` as two output formats?

---

Sounds good. I've made the change alongside the Makefile, which calls this script.